.github/workflows/komodod_cd.yml

```diff
@@ -109,11 +110,14 @@ jobs:
           DEBIAN_FRONTEND: noninteractive
         run: |
           sudo apt-get update  # prevents repo404 errors on apt-remove below
-          sudo apt-get remove php5.6-fpm php7.0-fpm php7.1-fpm php7.2-fpm php7.3-fpm php7.3-common php7.4-fpm
+          sudo apt-get remove php* msodbcsql17 mysql* powershell containernetworking-* containers* dotn*
           sudo apt-get update
           sudo apt-get upgrade -y
-          sudo apt-get install build-essential pkg-config libc6-dev m4 g++-multilib autoconf libtool libncurses-dev unzip git python zlib1g-dev wget bsdmainutils automake libboost-all-dev libssl-dev libprotobuf-dev protobuf-compiler libqrencode-dev libdb++-dev ntp ntpdate nano software-properties-common curl libevent-dev libcurl4-gnutls-dev cmake clang libsodium-dev -y
-          sudo apt-get install build-essential pkg-config libc6-dev m4 g++-multilib autoconf libtool ncurses-dev unzip git python python-zmq zlib1g-dev wget libcurl4-gnutls-dev bsdmainutils automake curl cmake mingw-w64
+          sudo apt-get install build-essential pkg-config libc6-dev m4 g++-multilib autoconf libtool \
+                               libncurses-dev unzip wget bsdmainutils automake libboost-all-dev libssl-dev \
+                               libprotobuf-dev protobuf-compiler libqrencode-dev libdb++-dev ntp ntpdate nano \
```

Comment:
Unfamiliar with CI flow, but maybe we want to take a second look at these? For example, how could nano be a dep?

Reply:
I believe better cd workflow was in research branch, will recheck and PR updates later.
Also, we are not really using it as far as i'm aware. Still, no reason to keep it messy, yeah.